src/hotspot/cpu/aarch64/gc/g1/g1BarrierSetAssembler_aarch64.hpp

```diff
@@ -46,6 +46,7 @@ class G1BarrierSetAssembler: public ModRefBarrierSetAssembler {
                             Register pre_val,
                             Register thread,
                             Register tmp,
```

Comment:
It would be better to use `tmp1` as the definition site does.

Reply:
👍 Fixed. Made all two temporary register names consistent across g1BarrierSetAssembler